src/metabase/models/table.clj

```diff
@@ -57,9 +57,8 @@
 
 (t2/define-before-insert :model/Table
   [table]
-  (let [defaults {:display_name        (humanization/name->human-readable-name (:name table))
-                  :field_order         (driver/default-field-order (t2/select-one-fn :engine Database :id (:db_id table)))
-                  :initial_sync_status "incomplete"}]
```

Comment:
We never needed to set a default here for `initial_sync_status`. It should be set by the caller of `t2/insert-` for clarity